std_msgs/Header header
ObjectDesc[] objects
